eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -340,6 +340,42 @@
         "code": "java.method.added",
         "new": "method com.azure.core.http.policy.HttpLogOptions com.azure.core.http.policy.HttpLogOptions::setResponseLogger(com.azure.core.http.policy.HttpResponseLogger)",
         "justification": "New method included in azure-core."
+      },
+      {
+        "code": "java.annotation.attributeValueChanged",
+        "old": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
```

Comment:
There should be another pattern already suppression a bunch of JsonWebKey API additions, you could add this to that.